scrypto/src/component/stubs.rs

```diff
@@ -239,15 +240,16 @@ extern_blueprint_internal! {
         ) -> Global<MultiResourcePool>;
     },
     {
-        fn contribute(&mut self, buckets: Vec<Bucket>) -> (Bucket, Vec<Bucket>);
+        fn contribute(&mut self, buckets: Vec<FungibleBucket>)
+            -> (FungibleBucket, Vec<FungibleBucket>);
         fn redeem(&mut self, bucket: Bucket) -> Vec<Bucket>;
```

Comment:
I think redeem should be a `(FungibleBucket) -> Vec<FungibleBucket>` since the input is the pool units (fungible) and the output is fungible resources.